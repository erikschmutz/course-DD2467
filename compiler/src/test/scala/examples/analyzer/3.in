let a = 1
let b = a

(*====================
val a : int
val b : int
- :int
====================*)