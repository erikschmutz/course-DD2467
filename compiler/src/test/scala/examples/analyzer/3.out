AnalyzerResult(
    Integer(),
    Enviroment(
        List(
            EnviromentEntry(
                Identifier(a),Integer()
            ),
            EnviromentEntry(
                Identifier(b),
                Integer()
            )
        )
    )
)